README.md

```diff
@@ -820,7 +820,11 @@ Notes:
     - works only for *multi-gpu*.
     - currently only includes the version of *multipack* optimized for linear attention implementations like *flash-attn*.
  * Notes on Fast MoE
-    - `--fast_moe` is an integer value that configures the amount of expert parallel sharding (ep_degree).
+    - `--fast_moe` takes either an integer or boolean value.
+      - When its an integer it enables expert parallel sharding with the given expert parallel degree along with Scatter MoE kernels.
+      - When its boolean False, it is Scatter MoE Kernels with complete replication of experts across ranks.
+      - When its boolean True, it is Scatter MoE Kernels with experts sharded based on the top level sharding protocol (e.g. FSDP).
+      - When a boolean is passed, the expert parallel degree defaults to 1. When an integer is passed, Scatter MoE defaults to replicating experts across ranks.
```

Comment:
Maybe better to split the argument in two then 
1. When its boolean -
  the expert parallel degree defaults to 1. When an integer is passed, Scatter MoE defaults to replicating experts across ranks.
 1.1 If set to True, it is Scatter MoE Kernels with experts sharded based on the top level sharding protocol (e.g. FSDP).
 1.2 If set to False, it is Scatter MoE Kernels with complete replication of experts across ranks.
 
 2. When its integer - 
   If passed `n` enables expert parallel sharding with the `n` expert parallel degree along with Scatter MoE kernels <insert link here>.

Reply:
Modified, have a look!